### PR TITLE
update: Add default format setting (and add workbook closing)

### LINKED ISSUE #10: [Excel] Add the minimum required styles
current output:

<details>

image

</details>

At least, these styles should be added:

- [ ] column width (auto adjustment?)
- [ ] alignment (left top)
- [ ] ruled lines
- [ ] header style (bold, background color)

### PR DESCRIPTION
**Pull request by gmdr1024**
fix #10

image

### CODE REVIEW

**Comment by gmdr1024** [comment]
Supplement
Column width adjustment will be discussed in #42

File: src/data.rs [hunk 1]
```diff
@@ -250,12 +250,22 @@ impl Data {
...
+            .set_font_size(16.0)
+            .set_bold();
+        let head_row_format = workbook.add_format()
+            .set_text_wrap()
+            .set_align(FormatAlignment::CenterAcross)
+            .set_border(FormatBorder::Thin)
+            .set_bg_color(FormatColor::Cyan);
+        let data_row_format = workbook.add_format()
+            .set_text_wrap()
+            .set_border(FormatBorder::Thin);
```
Comment (ymtdzzz) [inline src/data.rs]: Set top alignment for vertical alignment (current: bottom alignment)

File: src/data.rs [hunk 2]
```diff
@@ -325,6 +335,7 @@ impl Data {
                 }
             }
         }
+        let _close_result = workbook.close();
```
Comment (ymtdzzz) [inline src/data.rs]: Use `?` op for error handling.

Change
```
let _close_result = workbook.close();
```
to
```
workbook.close()?;
```

**Review by ymtdzzz — CHANGES_REQUESTED** [review]
Great work, @gmdr1024 !
I've left some comments to improve codes, please fix them.

In addition, the build seems to be failed because of some format errors. Execute `cargo +stable fmt` command before commit your changes.

For more information, check the Contributing section of README.md.
https://github.com/toolbox-labo/md-design-doc#contributing

File: src/data.rs (hunk 1 above)
Comment (gmdr1024) [reply src/data.rs]: Done :)

File: src/data.rs (hunk 2 above)
Comment (gmdr1024) [reply src/data.rs]: Done :)

**Review by ymtdzzz — APPROVED** [review]
Thanks @gmdr1024 , LGTM

**Pull request merged**